src/Kunstmaan/MultiDomainBundle/Router/DomainBasedLocaleRouter.php

```diff
@@ -172,15 +156,19 @@ private function getReverseLocaleMap(): array
      */
     public function getRouteCollection(): RouteCollection
     {
-        if (($this->otherSite && $this->isMultiLanguage($this->otherSite['host'])) || (!$this->otherSite && $this->isMultiLanguage())) {
-            if (!$this->routeCollectionMultiLanguage) {
-                $this->routeCollectionMultiLanguage = new RouteCollection();
+        if (!$this->enabledImprovedRouter) {
+            trigger_deprecation('kunstmaan/multidomain-bundle', '7.2', 'Not enabling the improved router is deprecated and the changed and improved router will be the default in 8.0. Set the "kunstmaan_multi_domain.enable_improved_domain_based_local_router" config to true.');
```

Comment:
```suggestion
            trigger_deprecation('kunstmaan/multidomain-bundle', '7.2', 'Not enabling the improved router is deprecated and the changed and improved router will be the default in 8.0. Set the "kunstmaan_node.enable_improved_domain_based_local_router" config to true.');
```

The option is configured on the node bundle currently or is it added in the wrong bundle? And if it is the node-bundle config the config option itself is incorrect `enable_improved_router`

Reply:
I didn't get it working when adding the configuration option inside the multi domain bundle. Maybe it missed something, so I added it inside the node bundle.